inscriptions/ISic3097.xml

```diff
@@ -177,31 +182,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI331422">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">τῇ πρ</supplied>ὸ αʹ καλ
-                    <supplied reason="lost">ανδῶν</supplied>
-                    <lb n="2"/>
-                    <supplied reason="lost">Ἰενου</supplied>αρίων
-                    <supplied reason="lost">τῇ ὑπατίᾳ</supplied>
-                    <lb n="3"/>
-                    <supplied reason="lost">Ἀρκα</supplied>δίου
-                    <expan>
-                        <abbr>κ</abbr>
-                        <ex>αὶ</ex>
-                    </expan>
-                    ∙ Ὁν
-                    <supplied reason="lost">ωρίου τὸ εʹ</supplied>
-                    <lb n="4"/>
-                    <supplied reason="lost">— — — — — —</supplied>
-                    <unclear>Τ</unclear>
-                    <supplied reason="lost">— — — — —</supplied>
-                    <lb n="5"/>
+                    <lb n="1"/><supplied reason="lost">τῇ</supplied> <supplied reason="lost">πρ</supplied>ὸ <num value="1">α</num> καλ<supplied reason="lost">ανδῶν</supplied>
+                    <lb n="2"/><supplied reason="lost">Ἰενου</supplied>αρίων <supplied reason="lost">τῇ</supplied> <supplied reason="lost">ὑπατίᾳ</supplied>
+                    <lb n="3"/><supplied reason="lost">Ἀρκα</supplied>δίου <expan><abbr>κ</abbr><ex>αὶ</ex></expan> Ὁν<supplied reason="lost">ωρίου</supplied> <supplied reason="lost">τὸ</supplied> <supplied reason="lost"><num value="5">ε</num></supplied>
+                    <lb n="4"/><gap reason="lost" extent="unknown" unit="character"/><orig><unclear>Τ</unclear></orig><gap reason="lost" extent="unknown" unit="character"/>
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app>
+                        <note>Text after SEG</note> <!-- check editor -->
```

Comment:
Text is that of Ferrua, with drawing, fig. 1.f [so ignore the large 'f']:
![image](https://user-images.githubusercontent.com/25080386/104303922-9a0f1380-54c2-11eb-93ea-c794803b9efd.png)